### PR TITLE
feat(content-switcher): add size variants

### PR DESCRIPTION
**Pull request by kennylam**
### Related Ticket(s)

#528  

### Description

This adds size variants to `content-switcher`.

### Changelog

**New**

- `sm` and `xl` sizes

### CODE REVIEW

File: src/components/content-switcher/content-switcher.scss [hunk 1]
```diff
@@ -13,6 +13,14 @@ $css--plex: true !default;
   @extend .#{$prefix}--content-switcher;
 }
 
+:host(#{$prefix}-content-switcher[size='sm']) {
+  height: rem(32px);
```
Comment (asudoh) [inline src/components/content-switcher/content-switcher.scss:17]:
```suggestion
  @extend .#{$prefix}--content-switcher--sm;
```

File: src/components/content-switcher/content-switcher.scss [hunk 2]
```diff
@@ -13,6 +13,14 @@ $css--plex: true !default;
...
+}
+
+:host(#{$prefix}-content-switcher[size='xl']) {
+  height: rem(48px);
```
Comment (asudoh) [inline src/components/content-switcher/content-switcher.scss:21]:
```suggestion
  @extend .#{$prefix}--content-switcher--xl;
```

**Review by asudoh — COMMENTED** [review]
Thank you for jumping in @kennylam! Please let me know the change suggestion doesn't work.

File: src/components/content-switcher/content-switcher.scss (hunk 1 above)
Comment (kennylam) [reply src/components/content-switcher/content-switcher.scss]: @asudoh Good call! Interesting that this works, in the past I've had problems extending classes directly from other imports.

File: src/components/content-switcher/content-switcher.scss (hunk 2 above)
Comment (kennylam) [reply src/components/content-switcher/content-switcher.scss]: @asudoh Good call! Interesting that this works, in the past I've had problems extending classes directly from other imports.

Comment (asudoh) [reply src/components/content-switcher/content-switcher.scss]: @kennylam Yeah it has to be a simple selector I believe but where it works we can avoid CSS rule duplicates.

**Review by asudoh — APPROVED** [review]
Thank you @kennylam for the update!

**Pull request merged**